lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -35,150 +36,110 @@
 | Preparation
 |
 @preparation{
-- The kinesthetic activity in this lesson requires a ruler and 4-8 pennies for each group of 3. 
+- During the kinesthetic activity in this lesson, you will split your students into groups of 3. Each group will need a ruler, a marker, tape, and 5 pennies.
 - There is an optional activity in this lesson that would require you to print an extra data cycle for your students.
-- In each of the first 3 Investigate sections of this lesson, students will calculate a Measure of Center from a list. If you would like to be able to direct students to printed versions with space for them to write, make copies of @opt-printable-exercise{mean-median-modes.adoc}.
-- If you are using our Google Slides, adjust them based on which of these activities you will be doing with your students.
+- The first section of the lesson is an introduction to measures of center. If you are confident that your students have a solid understanding of mean, median, and mode, you can jump to the second section of the lesson, where students will use @proglang to compute measures of center, and then consider which measure of center is the most appropriate.
+- If you are using our Google Slides, adjust the slides based on which portions of the lesson you will be doing with your students.
 }
 
 |===
 
-== Mean @duration{15 minutes}
+== Introduction to Measures of Center @duration{15 minutes}
```

Comment:
Apropos of our "lesson section naming" discussion on Wednesday, this section title could be improved. I'd vote for either "What Value is Typical?" or just "Mean, Median, and Modes".

The current title adds zero information for teachers: they already know the first section is probably the introduction, and they already know the lesson will be about Measures of Center.